shttps/Parsing.cpp

```diff
@@ -96,19 +95,21 @@ namespace shttps {
                 {"snd",     {"audio/basic"}},
                 {"mp3",     {"audio/mpeg"}},
                 {"mp4",     {"audio/mp4"}},
-                {"ogg",     {"audio/ogg",                                                         "video/ogg", "application/ogg"}},
-                {"ogv",     {"video/ogg",                                                         "application/ogg"}},
-                {"wav",     {"audio/wav",                                                         "audio/wav", "audio/x-wav", "audio/x-pn-wav"}},
+                {"ogg",     {"application/ogg", "audio/ogg", "video/ogg"}},
+                {"ogv",     {"video/ogg", "application/ogg"}},
+                {"oga",     {"audio/ogg", "application/ogg"}},
+                {"wav",     {"audio/wav", "audio/wav", "audio/x-wav", "audio/x-pn-wav"}},
                 {"aif",     {"audio/x-aiff"}},
                 {"aiff",    {"audio/x-aiff"}},
                 {"aifc",    {"audio/x-aiff"}},
                 {"mid",     {"audio/x-midi"}},
                 {"midi",    {"audio/x-midi"}},
                 {"mp2",     {"audio/x-mpeg"}},
-                {"wrl",     {"model/vrml",                                                        "x-world/x-vrml"}},
+                {"wrl",     {"model/vrml", "x-world/x-vrml"}},
                 {"ics",     {"text/calendar"}},
-                {"csv",     {"text/comma-separated-values",                                       "text/plain"}},
-                {"tsv",     {"text/tab-separated-values",                                         "text/plain"}},
+                {"csv",     {"text/comma-separated-values", "text/plain", "application/vnd.ms-excel",
```

Comment:
Currently the `/knora.json` route returns `text/comma-separated-values`  for a CSV file, but this MIME type is unregistered. The registered MIME type for CSV is `text/csv`: https://www.rfc-editor.org/rfc/rfc4180.html